apps/mull-api/src/app/entities/location.entity.ts

```diff
@@ -9,7 +9,7 @@ export class Location implements ILocation {
   id: number;
 
   @Column('point')
```

Comment:
This will set the type to varchar(255) in the database
```suggestion
  @Column()
```